doc/introduction.rst

```diff
@@ -133,6 +133,10 @@ alien a small number of pixels every frame will cause it to slide across the
 screen. Once it slides off the right-hand side of the screen, we reset it back
 to the left.
 
+Your Functions draw() and update() work in similar ways but are designed for two different purposes.
```

Comment:
"functions" should be lowercase.

Please put double-backticks "``" around `draw()` and `update()` so that they appear as code.